ui/packages/consul-ui/app/components/consul/auth-method/search-bar/index.hbs

```diff
@@ -0,0 +1,158 @@
+          <dd>{{value}}</dd>
+        </dl>
+      </search.RemoveFilter>
+{{/let}}
+
+    </:status>
+    <:search as |search|>
+      <search.Search
+        @onsearch={{action @onsearch}}
+        @value={{@search}}
+        @placeholder={{t "common.search.search"}}
+      >
+        <search.Select
+          class="type-search-properties"
+          @position="right"
+          @onchange={{action @filter.searchproperty.change}}
+          @multiple={{true}}
+          @required={{true}}
+        as |components|>
+          <BlockSlot @name="selected">
+            <span>
+              {{t "common.search.searchproperty"}}
+            </span>
+          </BlockSlot>
+          <BlockSlot @name="options">
+    {{#let components.Optgroup components.Option as |Optgroup Option|}}
+            {{#each @filter.searchproperty.default as |prop|}}
+              <Option @value={{prop}} @selected={{contains prop @filter.searchproperty.value}}>
+                {{t (concat "common.consul." (lowercase prop))}}
+              </Option>
+            {{/each}}
+    {{/let}}
+          </BlockSlot>
+        </search.Select>
+      </search.Search>
+    </:search>
+    <:filter as |search|>
+      <search.Select
+        class="type-kind"
+        @position="left"
+        @onchange={{action @filter.kind.change}}
+        @multiple={{true}}
+      as |components|>
+        <BlockSlot @name="selected">
+          <span>
+            {{t "components.consul.auth-method.search-bar.kind.name"}}
+          </span>
+        </BlockSlot>
+        <BlockSlot @name="options">
+  {{#let components.Optgroup components.Option as |Optgroup Option|}}
+    {{#each (array "kubernetes" "oidc" "jwt") as |type|}}
+          <Option class="{{type}}" @value={{type}} @selected={{contains type @filter.kind.value}}>
+            {{t (concat "components.consul.auth-method.search-bar.kind.options." type)
+                default=(array
+                  (concat "common.brand." type)
+                )
+            }}
+          </Option>
+    {{/each}}
+  {{/let}}
+        </BlockSlot>
+      </search.Select>
+      <search.Select
+        class="type-locality"
+        @position="left"
+        @onchange={{action @filter.source.change}}
+        @multiple={{true}}
+      as |components|>
+        <BlockSlot @name="selected">
+          <span>
+            {{t "components.consul.auth-method.search-bar.locality.name"}}
+          </span>
+        </BlockSlot>
+        <BlockSlot @name="options">
+      {{#let components.Optgroup components.Option as |Optgroup Option|}}
+        {{#each (array "local" "global") as |option|}}
+          <Option class="{{option}}" @value={{option}} @selected={{contains option @filter.types}}>
+            Creates
+            {{t (concat "components.consul.auth-method.search-bar.locality.options." option)
+                default=(array
+                  (concat "common.brand." option)
+                )
+            }}
+            tokens
```

Comment:
Couple of things here:

No big deal but you can probably take this default out, locality values will never be the name of a brand.

More importantly, the entire phrase should use the `t` helper so replace `Creates -- tokens` should be replaced with `t`. Then in the translation file for this key specify the full human phrase there (`Creates --- tokens`).

Lemme know if you don't follow general the thinking here and we can chat through in more detail.